### ISSUE TITLE
Skip Issue

### ISSUE THREAD

**Issue by Qarzival** [OriginalAlien/TikTokBotter#3]
Managed to run the script successfully, and it loads perfectly with the exception of this issue. 
It skips all the "active send pages"  from zefoy and I can't seem to figure out why? ([Here's a screenshot of the issue](https://ibb.co/mBYsrfs))
**Besides that, thanks for sharing this amazing script, truly appreciate it.** 

- Chrome version= 100.0.48...
- Python version= 3.6.0
- Chrome WebDriver version = 100

**Comment by OriginalAlien** [comment]
Yeah zefoy updated the xpath which finds where to click and since it isn't the new XPath it gives the error and skipped it
I'm currently updating it and adding new features as well

**Comment by Qarzival** [comment]
@OriginalAlien Thanks for the feedback, can't wait for the new features you're working on. I'm a bit rusty with python. I'd have loved to help out in anyway I can.

**Comment by OriginalAlien** [comment]
fixed it, it should work now

**Comment by Qarzival** [comment]
Great, just tested it and it works fine. 
Though the following lines of code have to first be edited before running smoothly:
```
118. service = Service(executable_path='C:\\Users\\USER\\OneDrive\\Desktop\\chromedriver.exe')
119. driver = webdriver.Chrome(service=service, options=option)
```
or else, users will run into the following error:
```
[>] Error: __init__() got an unexpected keyword argument 'service'
[>] Press Enter to Exit
```
Please replace with the original:
`driver = webdriver.Chrome("INSERT \\PATH \\TO \\WEB \\CHROME DRIVER.EXE", options=option) #Put chromedriver.exe file loction between quotation marks using double back slashes.`

Besides that, thanks for the update @OriginalAlien. 
Truly appreciate your talent once again.
